ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2759,6 +2861,15 @@ AHU_RTC_RHC_FDPM_REFM_WLDS:
   - REFM
   - SAFETY/WLDS
 
+AHU_RTC_RHC_WLDS:
```

Comment:
Assuming this device has OA directly, but no data for it? So itd be an incomplete version of some OA control AHU?

Reply:
Added OAFM to this type.